contracts/src/Teleporter/ITeleporterMessenger.sol

```diff
@@ -66,18 +66,14 @@ interface ITeleporterMessenger {
      * @dev Emitted when an additional fee amount is added to a Teleporter message that had previously
      * been sent, but not yet delivered to the destination chain.
      */
-    event AddFeeAmount(
-        bytes32 indexed destinationBlockchainID,
-        uint256 indexed messageID,
-        TeleporterFeeInfo updatedFeeInfo
-    );
+    event AddFeeAmount(bytes32 indexed messageID, TeleporterFeeInfo updatedFeeInfo);
 
     /**
      * @dev Emitted when a Teleporter message is being delivered on the destination chain to an address,
      * but message execution fails. Failed messages can then be retried with `retryMessageExecution`
      */
     event MessageExecutionFailed(
-        bytes32 indexed originBlockchainID, uint256 indexed messageID, TeleporterMessage message
+        bytes32 indexed messageID, bytes32 indexed originBlockchainID, TeleporterMessage message
```

Comment:
not sure if we'd care about `originBlockchainID` in this event and `MessageExecuted` if message ids are unique

Reply:
Yeah I thought about removing it entirely, but think it could still be helpful in tracking down the source chain message since that is not known for a given `messageID` even though the message ID is unique.